tcms/kiwi_auth/tests/test_views.py

```diff
@@ -199,6 +201,20 @@ def test_register_user_and_activate_by_admin(self):
                                 '<a href="mailto:{}">{}</a>'.format(email, name),
                                 html=True)
 
+    def test_invalid_form(self):
+        response = self.client.post(
+            self.register_url,
+            {
+                "username": "kiwi-tester",
+                "password1": "password-1",
+                "password2": "password-2",
+                "email": "new-tester@example.com",
+            },
+            follow=False,
+        )
+        self.assertEqual(response.status_code, 200)
```

Comment:
there should be an error message shown telling the user which field is invalid/has error. Assert on it.